integration/mock-googleapi/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
I probably wouldn't have created a helm chart for the mock, and just used a simple deployment yaml but I'm happy to leave it I guess.

Reply:
I just find it easy to redeploy or update when I have already running test env. that's all